scripts/wrong-filename.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```

Comment:
```suggestion
# SPDX-License-Identifier: MIT

```
It is suggested to have an SPDX license identifier at the top of the code especially for scripts to clarify usage.